nbodykit/plugins/datasource/ZeldovichSim.py

```diff
@@ -0,0 +1,96 @@
+from nbodykit.extensionpoints import DataSource
+from nbodykit import utils
+
+import numpy
+import logging
+
+logger = logging.getLogger('ZeldovichSim')
+         
+class ZeldovichSimDataSource(DataSource):
```

Comment:
that's an interesting idea...we'll certainly need to start increasing the synergy between FastPM and nbodykit once we get going on the emulator a bit more, so this is definitely worth thinking about. How much work would it be?